content/manager/manager-recovery.md

```diff
@@ -38,6 +38,10 @@ To create a snapshot, use the snapshot create command:
 cfy snapshots create -s my_snapshot
 {{< /gsHighlight >}}
 
+{{% gsNote title="Note" %}}
+To learn more about the snapshots CLI command, see [here](http://docs.getcloudify.org/api/#snapshots).
```

Comment:
the link is to the REST API docs.. it doesn't provide any info on the CLI command. The CLI command is exposed under cloudify-cli.readthedocs.org .. once we release the latest API to readthedocs.. which should be ASAP.